cypress/support/pages/signUp.pageObject.js

```diff
@@ -0,0 +1,48 @@
+import PageObject from '../PageObject';
+
+class SignUpPageObject extends PageObject {
+  url = '/#/register';
+
+  get usernameField() {
+    return cy.getByDataCy('username-sign-up');
+  }
+
+  get emailField() {
+    return cy.getByDataCy('email-sign-up');
+  }
+
+  get passwordField() {
+    return cy.getByDataCy('password-sign-up');
+  }
+
+  get signUpBtn() {
+    return cy.getByDataCy('sign-up-btn');
```

Comment:
Ensure that the element with data-cy attribute 'sign-up-btn' exists in the HTML. This is crucial for the `clickSignUpBtn` method to function correctly.